src/plugins/data/common/index_patterns/index_patterns/index_pattern.ts

```diff
@@ -266,12 +277,12 @@ export class IndexPattern implements IIndexPattern {
     }
 
     const savedObject = await this.savedObjectsClient.get(type, this.id);
-    this.version = savedObject._version;
 
     const response = {
-      _id: savedObject.id,
-      _type: savedObject.type,
-      _source: _.cloneDeep(savedObject.attributes),
+      id: savedObject.id,
+      type: savedObject.type,
```

Comment:
type is always 'index_pattern' so its not needed on the Spec, we can reconstruct index pattern without it